services/movie.py

```diff
@@ -0,0 +1,31 @@
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None, actors_ids: list[int] = None
+) -> list[Movie]:
+    filters = {}
+    if genres_ids:
+        filters["genres__id__in"] = genres_ids
+    if actors_ids:
+        filters["actors__id__in"] = actors_ids
+    return Movie.objects.filter(**filters)
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Using `Movie.objects.get(id=movie_id)` will raise a `DoesNotExist` exception if no movie with the given ID is found. Consider using `get_object_or_404` if this function is used in a view, or handle the exception appropriately to avoid unhandled exceptions.